Files/Views/ColumnShellPage.xaml.cs

```diff
@@ -280,26 +278,25 @@ private async void ColumnShellPage_TextChanged(ISearchBox sender, SearchBoxTextC
             }
         }
 
-        private async void ColumnShellPage_SuggestionChosen(ISearchBox sender, SearchBoxSuggestionChosenEventArgs e)
+        private async void ColumnShellPage_QuerySubmitted(ISearchBox sender, SearchBoxQuerySubmittedEventArgs e)
         {
-            if (e.SelectedSuggestion.PrimaryItemAttribute == StorageItemTypes.Folder)
+            if (e.ChosenSuggestion is ListedItem item)
             {
-                ItemDisplayFrame.Navigate(typeof(ColumnViewBase), new NavigationArguments()
+                if (item.PrimaryItemAttribute == StorageItemTypes.Folder)
                 {
-                    NavPathParam = e.SelectedSuggestion.ItemPath,
-                    AssociatedTabInstance = this
-                });
-            }
-            else
-            {
-                // TODO: Add fancy file launch options similar to Interactions.cs OpenSelectedItems()
-                await Win32Helpers.InvokeWin32ComponentAsync(e.SelectedSuggestion.ItemPath, this);
+                    ItemDisplayFrame.Navigate(typeof(ColumnViewBase), new NavigationArguments()
```

Comment:
Huh? There already exists a helper method that opens a folder or a file [here](https://github.com/files-community/Files/blob/818565547cf19d675c7daebbe284ec89a729ed6a/Files/Helpers/NavigationHelpers.cs#L126)